python/foglamp/services/core/api/plugin_discovery.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from aiohttp import web
+from foglamp.common.plugin_discovery import PluginDiscoveryInstalled
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_help = """
+    -------------------------------------------------------------------------------
+    | GET             | /foglamp/plugins/installed                                |
+    -------------------------------------------------------------------------------
+"""
+
+
+async def get_plugins_installed(request):
+    """ get list of installed plugins
+
+    :Example:
+        curl -X GET http://localhost:8081 /foglamp/plugins/installed
+        curl -X GET http://localhost:8081 /foglamp/plugins/installed?type=north
+    """
+
+    plugin_type = None
+    if 'type' in request.query and request.query['type'] != '':
+        plugin_type = request.query['type']
+
+    if plugin_type is not None and plugin_type not in ['north', 'south']:
+        raise web.HTTPNotFound(reason="Invalid plugin type. Must be 'north' or 'south'.")
+
+    if plugin_type is None:
+        plugins_list = PluginDiscoveryInstalled.get_plugins("north")
+        plugins_list_south = PluginDiscoveryInstalled.get_plugins("south")
+        plugins_list.extend(plugins_list_south)
```

Comment:
well! PluginDiscoveryInstalled.get_plugins should have default value as None and return south+ north by default.  if we plan to make 2 calls; we  should name   plugins_list to plugins_list_north and then merge south+north into plugins_list

Reply:
done